.circleci/config.yml

```diff
@@ -205,6 +209,18 @@ jobs:
           paths:
             - usabillabv_php-http.tar
             - build-http.tags
+  build-prometheus-exporter-file:
+    machine: true
+    steps:
+      - checkout
+      - run: make build-prometheus-exporter-file
+      - run: cat ./tmp/build-prometheus-exporter-file.tags | xargs -I % docker inspect --format='%={{.Id}}:{{index .ContainerConfig.Env 1}}' %
```

Comment:
Would it be useful to give the path as an parameter to the make target? Now it sort of comes out of nowhere. Or we could make it a variable inside the makefile and pour this command in a make target as well.

Reply:
I've created an issue for this one: https://github.com/usabilla/php-docker-template/issues/101 :+1: